nizarmah/nizarmah.me
# Adds default Gatsby starter website with Netlify CMS integration

This pull request adds the following:
- Gatsby starter template website
- Netlify CMS integration

**Testing instructions**:

1. Make sure sandbox loads with default Gatsby page
2. Make sure that you can login into the Netlify CMS

**Author notes and concerns**:

1. The netlify cms settings contains some collections just so it isn't empty.

**Reviewers**
